cmake/JoinPaths.cmake

```diff
@@ -0,0 +1,22 @@
+# This module provides function for joining paths
+# known from most languages
+#
+# SPDX-License-Identifier: (MIT OR CC0-1.0)
```

Comment:
A note for reference: Here's a link to the CC0-1.0 license: https://creativecommons.org/publicdomain/zero/1.0/